programming/minibot-assembly-tutorial.md

```diff
@@ -0,0 +1,71 @@
+Creating a minibot is great for testing small projects. It's an easy change to try out new parts out and costs considerably less than a larger robot. 
```

Comment:
The document should start with a top-level header, i.e.

```md
# Assembling Minibots
```

Reply:
Okay! I changed the formatting.

Reply:
Headings are used for the document title, as well as for titles of sections within the document. We don't need the second two headings here because they are redundant. Additionally, there is not a reason to change heading levels more than one at a time i.e. from top-level header to 3rd level header